util/cron/test-linux64-gcc49.bash

```diff
@@ -1,19 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
So why was this file removed?

Reply:
We stopped testing with gcc 4.9 because LLVM requires at least 5.1 to build, and our 4.9 environment doesn't work with llvm that was built with 5.x.